react-frontend/src/components/Charts.tsx

```diff
@@ -89,6 +142,9 @@ function Charts({
 		if (cleanInput.indexOf('root of ') > -1) {
 			cleanInput.replace(/root\sof\s(.*)/g, 'sqrt($1)');
 		}
+		if (cleanInput.indexOf('_') > -1) {
```

Comment:
convergence of Wolfram result formatting and mathjs/Mathjax syntax. Wolfram indicates subscripts with underscore but the js expects subscripts to be in square brackets.